tests/build_feedstock_test.py

```diff
@@ -53,14 +54,12 @@ def test_build_feedstock_failure(mocker, capsys):
         return_value=False
     )
     mocker.patch(
-        'os.system',
-        side_effect=(lambda x: helpers.validate_cli(x, expect=["conda-build"], retval=1)) # Retval is 1 to simulate a failure.
+        'build_feedstock.build_feedstock',
```

Comment:
I am checking if this can be replaced with any other check to detect conda_build.api.build failure.

Reply:
Updated to mock `conda_buld.api.build` to throw a `ValueError` to simulate a failure.   Thanks @bethune-bryant  for suggesting this approach.